server/services/store/sqlstore/migrate.go

```diff
@@ -213,6 +225,194 @@ func (s *SQLStore) Migrate() error {
 	return engine.ApplyAll()
 }
 
+func (s *SQLStore) migrateSchemaVersion(migrations []*models.Migration) error {
```

Comment:
Naming is hard, but we are actually migrating the migration system. Can you update the naming accordingly and also add a few line of comments why this was needed etc.

Reply:
Done.